src/Operators/vorticity_operators.jl

```diff
@@ -9,7 +9,7 @@
 ##### See: https://github.com/CliMA/Oceananigans.jl/issues/1584
 #####
 
-@inline function Γᶠᶠᶜ(i, j, k, grid::ConformalCubedSphereFaceGrid, u, v)
+@inline function Γᶠᶠᶜ(i, j, k, grid::OrthogonalSphericalShellGrid, u, v)
```

Comment:
probably we are going to have to change this when we subdivide the face in multiple regions

Reply:
okie!